fix_homeworkk.py

```diff
@@ -0,0 +1,17 @@
+text = input("Write word or number:")
+if text.isdigit():
```

Comment:
This code is not related to this homework. 
This file should be deleted from this PR